redbox-core/redbox/storage/storage_handler.py

```diff
@@ -12,7 +12,7 @@ class BaseStorageHandler(ABC):
     """
 
     # dict comprehension for lowercase class name to class
-    model_type_map: ClassVar = {v.__name__.lower(): v for v in [Chunk, File]}
+    model_type_map: ClassVar = {v.__name__.lower(): v for v in [File]}
```

Comment:
this makes me wonder what the point of storing the file is at all.... what information does it hold that django doesnt already know about?

Reply:
it records the ingest status and in future likely more detail on that (which tools are ready, which version of the ingest process a file went through), that feels like it makes sense in core-api rather than then user facing app